ntbs-service/Pages/Notifications/Edit/MDRDetails.cshtml

```diff
@@ -0,0 +1,123 @@
+@page "/Notifications/{id:int}/Edit/MDRDetails/{handler?}"

```

Comment:
@jasiekmiko 
Assuming this PR gets merged in first - we're going to have to make sure this route is updated to reflect auto-binding

Reply:
Will get picked up by tests so not a massive concern

Reply:
Good spot!
I think we just need to change the variable name `id` to `NotificationId` - so that's a change that can be done preemptively :)